liiatools/datasets/social_work_workforce/csww_main_functions.py

```diff
@@ -0,0 +1,22 @@
+from liiatools.datasets.social_work_workforce.sample_data import generate_sample_csww_file
+from liiatools.csdatatools.util.stream import consume
+from liiatools.csdatatools.util.xml import etree, to_xml
+
+
+def generate_sample(output: str):
+    """
+    Export a sample file for testing
+
+    :param output: string containing the desired location and name of sample file
+    :return: .xml sample file in desired location
+    """
+
+    stream = generate_sample_csww_file()
+    builder = etree.TreeBuilder()
+    stream = to_xml(stream, builder)
+    consume(stream)
+
+    element = builder.close()
+    element = etree.tostring(element, encoding='utf-8', pretty_print=True)
+    with open(output, 'wb') as FILE:
```

Comment:
There doesn't seem to be any validation on the path before we use it, or any handling of errors that could result. For instance, if I run this with the -o flag like this: `liiatools csww generate-sample --o 1/2/3` it will cause an exception. Ideally, you'd want to catch something like that and fail more gracefully with some message like "The File Path provided is incorrect" or similar

Reply:
Have made some changes to catch this now